### PR TITLE
backend: init users and books apps + models

### PR DESCRIPTION
**Pull request by A-Seyfi**
Created two apps with their respective models: users, books

### CODE REVIEW

File: .env.example
Comment (Wolbyte) [inline .env.example:1]: You shouldn't delete this file, please restore it.

File: backend/users/models.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from books.models import Book
+
+# Create your models here.
+
+class Student(models.Model):
+    Name = models.CharField(max_length = 40)
```
Comment (Wolbyte) [inline backend/users/models.py]: Split this field into two, `first_name` and `last_name`, each with a max length of 20

File: backend/users/models.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    ClassNo = models.IntegerField()
```
Comment (Wolbyte) [inline backend/users/models.py]: Rename this field to `class_number` and set its max length to `4`

File: backend/users/models.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    NCode = models.IntegerField()
```
Comment (Wolbyte) [inline backend/users/models.py]: Rename this field to `national_code` and set its max length to 10

File: backend/users/models.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from books.models import Book
+
+# Create your models here.
```
Comment (Wolbyte) [inline backend/users/models.py]: Remove this line

File: backend/books/models.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+# Create your models here.
```
Comment (Wolbyte) [inline backend/books/models.py]: remove this line

File: backend/books/models.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+class Book(models.Model):
+    ISBN = models.CharField(max_length = 8)
```
Comment (Wolbyte) [inline backend/books/models.py]: change the `max_length` to 13

**Review by Wolbyte — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Please use `snake_case` to declare fields inside models

**Comment by A-Seyfi** [comment]
> Thank you for your contribution! Please use `snake_case` to declare fields inside models

**Review by Wolbyte — APPROVED** [review]
LGTM

**Pull request merged**